service/src/main/java/bio/terra/tanagra/app/configuration/AuthenticationConfiguration.java

```diff
@@ -15,10 +15,10 @@
     name = "Authentication",
     markdown =
         "Configure the authentication model.\n\n"
-            + "There are four separate flags that control which model is used: "
-            + "`tanagra.auth.disableChecks`, `tanagra.auth.iapGkeJwt`, `tanagra.auth.iapAppEngineJwt`, `tanagra.auth.bearerToken`. "
+            + "There are five separate flags that control which model is used: "
+            + "`tanagra.auth.disableChecks`, `tanagra.auth.iapGkeJwt`, `tanagra.auth.iapAppEngineJwt`, `tanagra.auth.bearerToken`, `tanagra.auth.auth0Jwt`. "
```

Comment:
`tanagra.auth.auth0Jwt` seems like a good name for the flag. It doesn't match the field names below though. Can you please update the flags below to `tanagra.auth.auth0Jwt.___` to match here?

Reply:
updated to `tanagra.auth.unverifiedJwt`